gauntlet/packages/gauntlet-serum-multisig/src/commands/multisig.ts

```diff
@@ -0,0 +1,180 @@
+import { Result } from '@chainlink/gauntlet-core'
+import { TransactionResponse, SolanaCommand, RawTransaction } from '@chainlink/gauntlet-solana'
+
+import { logger, BN } from '@chainlink/gauntlet-core/dist/utils'
+import { PublicKey, SYSVAR_RENT_PUBKEY, Keypair } from '@solana/web3.js'
+import { CONTRACT_LIST, getContract } from '@chainlink/gauntlet-solana-contracts'
+import { Idl, Program } from '@project-serum/anchor'
+
+export const wrapCommand = (command) => {
+  return class Multisig extends SolanaCommand {
+    command: SolanaCommand
+    program: Program<Idl>
+    proposal: PublicKey
+    multisigAddress: PublicKey
+    multisigSigner: PublicKey
+    owners: [PublicKey]
+    threshold: number
```

Comment:
Let's try to reduce the amount of context variables. I'd only leave `command`, `program` and `multisigAddress`